libs/hdf-converters/package.json

```diff
@@ -62,6 +64,7 @@
     "@types/jest": "^27.0.0",
     "@types/lodash": "^4.14.161",
     "@types/node": "^20.1.0",
+    "@types/validator": "^13.11.10",
```

Comment:
After discussion, it seemed like these needed to be in dependencies, so let's get rid of the double import that's happening here.

Reply:
Done.